### PR TITLE
Restructure JSON -azure

### PR DESCRIPTION
**Pull request by rajeevr2715**
**Requirement** 

Restructure azure json in installer vars.
Fixed issue in UI job and added create-serverless-service trigger job

### CODE REVIEW

File: features/extensions/azure/install.py [hunk 1]
```diff
@@ -83,30 +83,46 @@ def install(jazz_stackprefix, jenkins_url, jenkins_user, jenkins_api_token, jazz
...
+    azureAccount = OrderedDict()
+    azureAccount['ACCOUNTID'] = azure_subscription_id
+    azureAccount['ACCOUNTNAME'] = 'azureaccount'
+    azureAccount['SUBSCRIPTION_ID'] = 'AZ_SUBSCRIPTIONID'
+    azureAccount['CLIENT_ID'] = 'AZ_CLIENTID'
+    azureAccount["PASSWORD"] = 'AZ_PASSWORD'
+    azureAccount["TENANT_ID"] = 'AZ_TENANTID'
+    azureRegions = OrderedDict()
+    azureRegions['LOCATION'] = azure_location
+    azureRegions['REGION'] = azure_location.lower().replace(' ','')
```
Comment (raghits) [inline features/extensions/azure/install.py]: Azure Regions have spaces, isn't removing them going to be an issue?

File: features/extensions/azure/install.py [hunk 2]
```diff
@@ -83,30 +83,46 @@ def install(jazz_stackprefix, jenkins_url, jenkins_user, jenkins_api_token, jazz
...
+    azureRegions['APIM'] = OrderedDict()
+    azureRegions['APIM']['DEV'] = getTerraformOutputVar("dev_apim", extension_base)
+    azureRegions['APIM']['PROD'] = getTerraformOutputVar("prod_apim", extension_base)
+    azureRegions['APIM']['STG'] = getTerraformOutputVar("stage_apim", extension_base)
+    azureAccount['REGIONS'] = [azureRegions]
+    azureConfig['ACCOUNTS'] = [azureAccount]
+    azureDefault = OrderedDict()
+    azureDefault['ACCOUNTID'] = azure_subscription_id
+    azureDefault['PROVIDER'] = 'azure'
+    azureDefault['REGION'] = azure_location.lower().replace(' ','')
```
Comment (raghits) [inline features/extensions/azure/install.py]: Azure Regions have spaces, isn't removing them going to be an issue?

File: features/extensions/azure/install.py [hunk 3]
```diff
@@ -83,30 +83,46 @@ def install(jazz_stackprefix, jenkins_url, jenkins_user, jenkins_api_token, jazz
...
-    azureConfig['RESOURCE_GROUPS']['PRODUCTION'] = getTerraformOutputVar("prod_resource_group", extension_base)
-    azureConfig['APIM'] = OrderedDict()
-    azureConfig['APIM']['DEVELOPMENT'] = getTerraformOutputVar("dev_apim", extension_base)
-    azureConfig['APIM']['STAGING'] = getTerraformOutputVar("stage_apim", extension_base)
-    azureConfig['APIM']['PRODUCTION'] = getTerraformOutputVar("prod_apim", extension_base)
-
+    azureAccount = OrderedDict()
+    azureAccount['ACCOUNTID'] = azure_subscription_id
+    azureAccount['ACCOUNTNAME'] = 'azureaccount'
+    azureAccount['SUBSCRIPTION_ID'] = 'AZ_SUBSCRIPTIONID'
```
Comment (devsatishm) [inline features/extensions/azure/install.py]: `ACCOUNTID` & `SUBSCRIPTION_ID` will contain the same value (We've planned to use `ACCOUNTID` just to be consistent with other cloud provider (`aws`) that we use). Why are these two being assigned a bit differently in this code?

File: features/extensions/azure/install.py (hunk 1 above)
Comment (devsatishm) [reply features/extensions/azure/install.py]: I see the reason for this. Azure has `DisplayName` & `Name` for each region. Ex: `DisplayName` -> `East US 2` & `Name` -> `eastus2`. So, `azureRegions['LOCATION'] ` & `azureRegions['REGION']` are being used to store these values respectively (which makes sense).

@rajeevr2715  Is the user expected to specify `DisplayName` during installation?

File: features/extensions/azure/install.py [hunk 4]
```diff
@@ -83,30 +83,46 @@ def install(jazz_stackprefix, jenkins_url, jenkins_user, jenkins_api_token, jazz
...
+    azureRegions['RESOURCE_GROUPS']['PROD'] = getTerraformOutputVar("prod_resource_group", extension_base)
+    azureRegions['RESOURCE_GROUPS']['STG'] = getTerraformOutputVar("stage_resource_group", extension_base)
+    azureRegions['APIM'] = OrderedDict()
+    azureRegions['APIM']['DEV'] = getTerraformOutputVar("dev_apim", extension_base)
+    azureRegions['APIM']['PROD'] = getTerraformOutputVar("prod_apim", extension_base)
+    azureRegions['APIM']['STG'] = getTerraformOutputVar("stage_apim", extension_base)
+    azureAccount['REGIONS'] = [azureRegions]
+    azureConfig['ACCOUNTS'] = [azureAccount]
+    azureDefault = OrderedDict()
+    azureDefault['ACCOUNTID'] = azure_subscription_id
```
Comment (devsatishm) [inline features/extensions/azure/install.py]: Note - While this (assigning these values to `azureDefault` always) may work now, we need to revisit this piece of code when we add the ability to support multiple azure subscriptions & regions.

Comment (raghits) [reply features/extensions/azure/install.py]: Right. We will have to make some major changes on Azure Installer and Azure build packs to support multiple subscriptions/regions. We will address this then. There is no benefit in trying to remove it now. We just kept it there to be consistent with AWS.

**Pull request merged**